### DISCUSSION TITLE
episode plays view

### DISCUSSION THREAD

**Discussion by emhl** [IgnisDa/ryot#1043] [Ideas]
The History view for a series can be pretty overwhelming if you watched that series multiple times.
it would be awesome to see in the episode view: 
- when that episode was last viewed (not just if)
- if it has multiple plays when those happened

**Comment by IgnisDa** [comment]
> if it has multiple plays when those happened

This is already shown by the number alongside the image. 

How do you propose the UI for this should look like?
